extensions/wikia/AdEngine/js/lookup/a9.js

```diff
@@ -84,11 +123,13 @@ define('ext.wikia.adEngine.lookup.a9', [
 			priceMap = {};
 
 			slots = slotsContext.filterSlotMap(config[skin]);
-			a9Slots = Object.keys(slots).map(createSlotDefinition);
 
-			if (isVideoBidderEnabled()) {
-				a9Slots = a9Slots.concat(VIDEO_SLOTS.map(createVideoSlotDefinition));
-			}
+			var a9Slots = Object
```

Comment:
Is it possible that `a9Slots` will be empty? If so, should we proceed with execution?

Reply:
Handled 👍